tests/ord.test.js

```diff
@@ -0,0 +1,137 @@
+const cds = require("@sap/cds");
+const ord = require("../lib/ord");
+const path = require("path");
+
+describe("Tests for default ORD document", () => {
+  let csn;
+
+  beforeAll(async () => {
+    csn = await cds.load(path.join(__dirname, "bookshop", "srv"));
+  });
+
+  test("Successfully create ORD Documents with defaults", () => {
+    const document = ord(csn);
+    expect(document).toStrictEqual({
```

Comment:
I would not copy the entire output of the test in here. If you want to do a simple check that the result does not change without you noticing, you can use a jest snapshot instead. See https://jestjs.io/docs/snapshot-testing

Reply:
I wasn't aware of this feature. Very cool. I changed the test to use a snapshot.